helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -151,12 +154,22 @@ resource/logs:
       action: delete
     - key: k8s.namespace.name
       action: delete
+    - key: k8s.pod.labels.app
+      action: delete
     {{- range $_, $label := .Values.extraAttributes.podLabels }}
     - key: {{ printf "k8s.pod.labels.%s" $label }}
       action: delete
     {{- end }}
     {{- end }}
     {{- end }}
+
+transform/sck_fields:
+  log_statements:
+    - context: log
+      statements:
+        {{- if .Values.splunkPlatform.fieldNameConvention.renameFieldsSck }}
```

Comment:
You need to apply the condition to the whole receiver and its reverence in the pipeline

Reply:
I agree, but the reason why added it here is that if someone wants to add anything to the transform processor in the future, it would be easier to do. 
I have also made transform/logs in a similar fashion to resource/logs.

Reply:
If users want to add another processor and configure it, it's easy to define in the `agent.config` section. We should not add components with an empty body just to make them visible, it's a lot of different components available

Reply:
Yes, they can, but many customers will be migrating from SCK to SCK-Otel as SCK will be deprecated soon [https://github.com/splunk/splunk-connect-for-kubernetes#end-of-support].
That's why I'm adding this new processor. The only reason for adding a new processor is that to make SCK-Otel backward compatible with SCK. For other stuff, users can make use of `agent.config` as you suggested.

Reply:
Please don't add a new processor in the real production config if it's not being used by default. Just mention how to configure it in the migration doc